core/src/components/back-button/back-button.tsx

```diff
@@ -54,20 +54,32 @@ export class BackButton implements ComponentInterface, ButtonInterface {
    */
   @Prop() type: 'submit' | 'reset' | 'button' = 'button';
 
-  private get backButtonIcon() {
-    return this.icon != null ? this.icon : config.get('backButtonIcon', 'arrow-back');
+  get backButtonIcon() {
+    const icon = this.icon || config.get('backButtonIcon');
+    if (icon != null) {
+      // icon set on the component, next from the config
+      return icon;
+    }
+
+    if (this.mode === 'ios') {
+      // default md back button icon
+      return 'chevron-back';
+    }
+
+    // default md back button icon
+    return 'arrow-back-sharp';
   }
 
-  private get backButtonText() {
+  get backButtonText() {
```

Comment:
Why are we removing `private` from all of these?

Reply:
so i could unit test them. But doesn't actually matter and they're not made public since `@Method` is on them.